doozer/doozerlib/opm.py

```diff
@@ -26,19 +97,29 @@ async def verify_opm():
         raise IOError(f"opm version {version} is too old. Please upgrade to at least 1.47.0.")
 
 
-async def render(catalog: str, output_format: str = "yaml"):
+@retry(reraise=True, stop=stop_after_attempt(3), wait=wait_fixed(5))
+async def render(*input: str, output_format: str = "yaml", migrate: bool = False,
```

Comment:
```suggestion
async def render(*args: Tuple[str], output_format: str = "yaml", migrate: bool = False,
```
minor style suggestion

Reply:
also raise error if input is empty?

Reply:
My linter errs if I change the type to tuple... Thinking here we should declare the type of the element of the tuple.